Cabal/ChangeLog.md

```diff
@@ -773,7 +773,7 @@
   * Many checks added for common mistakes
   * New `--package-db=` option for specific package databases
   * Many internal changes to support cabal-install
-  * Stricter parsing for version strings, eg dissalows "1.05"
+  * Stricter parsing for version strings, eg disallows "1.05"
```

Comment:
Does this also catch the top level ones like `CONTRIBUTING.md`?

Reply:
It does now. I went with `find`, `grep` and `xargs` after being dissatisfied with the control over what files `typos` sees. I tried globs with its `[PATH]` option and its `--file-list <FILE_LIST>` option.